activesupport/lib/active_support/encrypted_configuration.rb

```diff
@@ -75,8 +75,8 @@ def require(*key)
       end
     end
 
-    # Find a upcased and double-underscored-joined string-version of the +key+ in ENV.
-    # Returns nil if the key isn't found or the value of default when passed If default is
+    # Find an upcased and double-underscored-joined string-version of the +key+ in ENV.
+    # Returns nil if the key isn't found or the value of default when passed. If default is
     # a block, it's called first.
```

Comment:
The whole comment is actually wrong here. The comment is talking about ENV which isn't used here.

I've got a PR with these and other comment fixes here: https://github.com/rails/rails/pull/56430

Reply:
I see it makes sense and apparently I was too short-sighted 🤦‍♂️ 

I'll be closing this one. Thanks!